controllers/dashboard.js

```diff
@@ -125,8 +125,8 @@ router.get('/', (req, res, next) => {
 					dueDate: null,
 				},
 				{
-					dueDate: {
-						$gte: ((new Date().getTime()) - 1000 * 60 * 60 * 24 * 7), // homeworks with max. 7 days before dueDate
+					dueDate: { // homeworks with max. 7 days after and 1 year before dueDate
```

Comment:
Hä? Ist der Kommentar jetzt nicht wieder genau verdreht?
$gte vor 7 Tagen & $lte in 1 Jahr => `// homeworks with dueDate before max. 7 days or max 1 year in the future`

Reply:
nevermind my logic is fucked up. You are right